peppy/project.py

```diff
@@ -512,6 +498,19 @@ def get_sample(self, sample_name):
 
         return samples[0]
 
+    def activate_subproject(self, subproject):
+        """
+        Activate a subproject.
+
+        This method will update Project attributes, adding new values
+        associated with the subproject indicated, and in case of collision with
+        an existing key/attribute the subproject's value will be favored.
+
+        :param str subproject: A string with a subproject name to be activated
+        :return Project: A Project with the selected subproject activated
+        """
+        return self.__init__(self.config_file, subproject=subproject)
```

Comment:
This should work, some attributes on a `Project` instance that aren't included in a config file that may have been dynamically set by a user post-construction would be reset by this. That may be the intended behavior...what do you guys think? If we wanted the update to be strictly based on the subproject, what about making this `self.parse_config_file(subproject)`?

Reply:
Also, this will return `None`. Is the goal to use a returned value, or just realize the effects of the call? If we just want the effects, we should remove `return` and just make this `self.__init__...` or `self.parse_config_file...`

Reply:
I didn’t even think about retaining the user-generated changes as I perceived subproject as a fresh start directly from the (slightly changed) config file. 
But you guys have broader experience with PEP usage in the real-life analyses, so maybe it is indeed beneficial to retain the changes.